velox/functions/prestosql/registration/ArrayFunctionsRegistration.cpp

```diff
@@ -145,6 +145,8 @@ void registerArrayFunctions(const std::string& prefix) {
 
   VELOX_REGISTER_VECTOR_FUNCTION(udf_array_sum, prefix + "array_sum");
   VELOX_REGISTER_VECTOR_FUNCTION(udf_repeat, prefix + "repeat");
+  VELOX_REGISTER_VECTOR_FUNCTION(
```

Comment:
This is not a Presto function. Do not register it here. Move this to Spark-specific registration code.

Please, document this function.